packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -338,6 +340,17 @@ QUnit.module('Chat', moduleConfig, () => {
             assert.strictEqual($(lastBubble).text(), text);
         });
     });
+
+    QUnit.module('focus', () => {
+        QUnit.test('The textarea element must be active after the Chat focus is invoked', function(assert) {
```

Comment:
```suggestion
        QUnit.test('The textarea input element must be active after the focus() method is invoked', function(assert) {
```

Reply:
JFYI: Also, input is active when 'dx-state-focus' class is presented on the root of component. Also we could check this class on the textArea element (to avoid thinking about shadow dom specifity). But, this way is also good.